docs/managing-advisories.md

```diff
@@ -0,0 +1,172 @@
+
+```sh
+wolfictl version
+```
+A successful installation of `wolfictl` will display output similar to the following. Note that the exact output will vary over time as new `wolfictl` versions are released. 
+
+```Output
+ __        __   ___    _       _____   ___    ____   _____   _
+ \ \      / /  / _ \  | |     |  ___| |_ _|  / ___| |_   _| | |
+  \ \ /\ / /  | | | | | |     | |_     | |  | |       | |   | |
+   \ V  V /   | |_| | | |___  |  _|    | |  | |___    | |   | |___
+    \_/\_/     \___/  |_____| |_|     |___|  \____|   |_|   |_____|
+wolfictl: A CLI helper for developing Wolfi
+
+GitVersion:    devel
+GitCommit:     6c98dc69a559192575d085d87fd916d8281dd67d
+GitTreeState:  clean
+BuildDate:     2024-07-23T01:57:17
+GoVersion:     go1.22.5
+Compiler:      gc
+Platform:      darwin/arm64
+```
+
+In the next section, you will complete your local setup so you can begin using the `wolfictl` tool to work with advisories.
+
+## Cloning Package and Advisory Repositories
+
+Before you can interact with security advisories, the  `wolfictl` tool will need access to existing Wolfi package and advisory information.
+
+You will need to clone two additional repositories: the [`wolfi-dev/os`](https://github.com/wolfi-dev/os) repository and the [`wolfi-dev/advisories`](https://github.com/wolfi-dev/advisories) repository. Execute the following commands to clone each of these repositories to your machine, and then navigate to the `advisories` directory.
+
+```sh
+git clone git@github.com:wolfi-dev/os.git 
+git clone git@github.com:wolfi-dev/advisories.git && cd_
+```
+
+With `wolfictl` installed and these two repositories cloned locally, you are now ready to interact with the security advisory database.
+
+
+## Viewing Existing Advisories
+
+First we will take a look at the existing advisories issued for packages in Wolfi. Keep in mind that the results shown here, and on your own machine, are snapshots in time. You should regularly check for changes to the upstream repository as new packages and advisories are issued. 
+
+You will be using the `wolfi advisory list` command to view existing advisories. There are a variety of flags which you can append to assist in your search.
+* `-p` lists all advisories for a given package name.
+* `-V` lists all advisories for a given CVE ID, across all packages.
+* `-t` lists all advisories by their most recent event type.
+* `-c` lists all advisories by detected component type.
+* `--history` reports the full list of events for displayed advisories.
+
+You can combine multiple flags together to conduct a more granular search. For a complete listing of available command options, you can run the `wolfictl advisory list -h` command in your terminal.
+
+For example, let’s say you want to find advisories for the `glibc` package as well as the full history of these advisories. 
+
+```sh
+wolfictl advisory list -p glibc
```

Comment:
```suggestion
wolfictl advisory list -p glibc --history
```

(It looks like below you're using the history flag)

Reply:
I don't know where it went, I swear it was there before... thank you for pointing that out!